Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2602,12 +2602,12 @@ def __init__(self, outputs_prefix=None, outputs_key_field=None, outputs=None, in
         self.outputs_key_field = outputs_key_field
 
         self._outputs_key_field = None  # type: Optional[List[str]]
-        if isinstance(outputs_key_field, STRING_TYPES):
+        if outputs_key_field is None or outputs_key_field == '':
```

Comment:
wouldnt it be better to just check `if not outputs_key_field:` ?

Reply:
sure